RD Add sharing instructions. (#901)
ubvu/open-handbook: tools/researchdrive/cost_access.qmd

Once the projectfolder is created you can invite internal and external collaborators yourself.

#### Inviting internal and external collaborators

The owner of the projectfolder can create new accounts and grant access through the following steps.

##### Inviting a new user

After having logged into Research Drive, select in the top menu bar: Dashboard > User accounts (2nd tab) > Invite user.

![The Research Drive top menu bar](/public/research-drive-lint.png)

> Enter the initials and last name and the e-mail in the pop-up window, if available use the e-mail of their research institute or university.   

Reviewer: ```suggestion
Enter the initials and last name and the email in the pop-up window, if available use the e-mail of their research institute or university.   
```

All VU email accounts are automatically required to use Multi-Factor Authentication (MFA). This is most commonly done through the Tiqr or Azure app. **VU students** do not automatically have MFA activated on their account. They can set this up by visiting the IT ServiceDesk, bringing an official ID.

> Contributors from outside the VU will need to create an eduID account to log into Research Drive. An [instruction](https://servicedesk.surf.nl/wiki/spaces/WIKI/pages/134479930/RD+How+to+create+an+eduID+account) on how to do this can be found on SURF's Research Drive Wiki. Additionally, it is important to turn on the MFA step in the eduID account. This is a requirement to better protect the data stored in VU Research Drive.

Reviewer: ```suggestion
Contributors from outside VU Amsterdam will need to create an eduID account to log into Research Drive. An [instruction](https://servicedesk.surf.nl/wiki/spaces/WIKI/pages/134479930/RD+How+to+create+an+eduID+account) on how to do this can be found on SURF's Research Drive Wiki. Additionally, it is important to turn on the MFA step in the eduID account. This is a requirement to better protect the data stored in VU Research Drive.
```

##### Grant Access Right to the New User  

> When a new user has activated their account, you will receive an e-mail confirming this. At this point you can grant them access to your entire project folder or only to a specific subfolder / several subfolders. 

Reviewer: ```suggestion
When a new user has activated their account, you will receive an email confirming this. At this point you can grant them access to your entire project folder or only to a specific subfolder / several subfolders. 
```

> After each folder or document you will see the term "Shared". If you click on the term "Shared" you will see an overview of all the users who have access rights to this folder or document.  By using the search function and selecting a user you can grant various rights (Can view/Can edit/Custom permissions) to this user. If the user needs specific rights you can click on "Custom permissions".  

Reviewer: ```suggestion
After each folder or document you will see the term "Shared". If you click on the term "Shared" you will see an overview of all users who have access rights to this folder or document.  By using the search function and selecting a user you can grant various rights (Can view/Can edit/Custom permissions) to this user. If the user needs specific rights you can click on "Custom permissions".  
```

Options:  

1. Create (Grants the ability to create new documents or upload them into a folder).  
2. Edit (Grants the ability to make alterations to documents). 

> 3. Share (this enables the user to invite new users to RD and afterward this user can grant them access to the parts of you project folder they have access to. In general, we discourage granting these rights. If many people have share rights, it can become difficult to have an overview of which user has access to the data). 

Reviewer: ```suggestion
3. Share (this enables the user to invite new users to RD and afterward this user can grant them access to the parts of your project folder they have access to. In general, we discourage granting these rights. If many people have share rights, it can become difficult to have an overview of which user has access to the data). 
```
